### DISCUSSION TITLE
对于朝向的理解

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#170] [Ideas]
![image](https://user-images.githubusercontent.com/100075314/163672903-71d5b59b-77d6-411c-9125-e52687027eae.png)
图中的意思是可以不指定朝向吗

**Comment by liusy58** [comment]
如果你的攻击范围所有的都是沿着x轴和y轴的话，就可以不指定，但是会不会这样比较无聊 ;-)
